kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -283,4 +286,22 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
       timeoutChecker.scheduleWithFixedDelay(checkTask, interval, interval, TimeUnit.MILLISECONDS)
     }
   }
+
+  def parseSessionHandle(sessionHandleStr: String): SessionHandle = {
+    try {
+      val sessionHandleParts = sessionHandleStr.split("\\|")
+      require(
+        sessionHandleParts.size == 3,
+        s"Expected 4 parameters but found ${sessionHandleParts.size}.")
+
+      val handleIdentifier = new HandleIdentifier(
+        UUID.fromString(sessionHandleParts(0)),
+        UUID.fromString(sessionHandleParts(1)))
+      val protocolVersion = TProtocolVersion.findByValue(sessionHandleParts(2).toInt)
+      new SessionHandle(handleIdentifier, protocolVersion)
+    } catch {
+      case NonFatal(_) =>
+        throw KyuubiSQLException(s"Invalid $sessionHandleStr")
```

Comment:
Does it seem to hide the root cause here? Shall we remove the entire `try-catch` or wrap for reason